Packs/Core/Integrations/CortexPlatformCore/CortexPlatformCore.yml

```diff
@@ -1599,6 +1599,8 @@ script:
         description: |
           A JSON encoded string representing a list of asset groups to search for. (e.g., `["group1", "group2"]`).
         isArray: true
+      - name: asset_categories
```

Comment:
what about the code?

Reply:
https://github.com/demisto/content/blob/45b565a4e3419caa211d68ab677da88392de2e26/Packs/Core/Integrations/CortexPlatformCore/CortexPlatformCore.py#L1054